misc/services/camel/de-metas-camel-edi/src/main/java/de/metas/edi/esb/commons/DesadvSettings.java

```diff
@@ -54,7 +53,8 @@ public static DesadvSettings forReceiverGLN(
 			return DesadvSettings.builder().clearingCenter(ClearingCenter.MetasfreshInHouseV2).build();
 		}
 
-		final ClearingCenter clearingCenter = ClearingCenter.ofValue(Util.resolveProperty(context, clearingCenterProperty, MetasfreshInHouseV2.toString()));
+		final String defaultClearingCenter = Util.resolveProperty(context, DEFAULT_CLEARING_CENTER);
```

Comment:
as i wrote
> what we need **👉 in addition 👈** is way to deplare a not-hardcoded default in the properties-File

so we do need the hardcoded default as a fallbac, too
Otherwise we'd have to edit every properties file that might be oput there on production-systems

Reply:
ah, nvm, i think that's what you did, right..now looking at the complete file..

Reply:
pushed